SwiftWisdomTests/DoubleTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DoubleTests.swift
+//  SwiftWisdom
+//
+//  Created by Stephen Wong on 5/1/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import XCTest
+import SwiftWisdom
+
+class DoubleTests: XCTestCase {
+
+    func testRounding() {
+        let unroundedDoubles = [M_PI, 5.0156, 10.014, 9999.994]
```

Comment:
maybe perhaps also do rounded to 5 decimal places(whereby you should still get the same value back)?